Shared/Test-IanaTimeZoneMapping.ps1

```diff
@@ -0,0 +1,83 @@
+﻿# Copyright (c) Microsoft Corporation.
```

Comment:
What is the reason for this being in the public `Shared`? Right now, it looks like we are only going to be using this in Health Checker, so let's just keep this there. 

Could even have this all within the Analyzer section of code.

Reply:
Test-IanaTimeZoneMapping.ps1 was added as shared function because we plan to publish the script mentioned in the KB article on CSS-Exchange to make it easier for customers to download and run the script.

Reply:
For now, let's put it under Health Checker, if that script is created and released, at that time we can move this out to the `Shared`

Reply:
Done